### PR TITLE
docs: Add send 1-to-many message section to README

### PR DESCRIPTION
**Pull request by jhaaaa**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,6 +184,55 @@ const conversation = await xmtp.conversations.newConversation(
 await conversation.send('Hello world')
 ```
 
+#### Send a 1:many message
```
Comment (fabriguespe) [inline README.md:187]: Broadcasting messages

File: README.md [hunk 2]
```diff
@@ -320,7 +320,56 @@ for (const conversation of myAppConversations) {
...
+    //Checking the activation status of each wallet
+    const wallet = broadcasts_array[i]
+    const canMessage = broadcasts_canMessage[i]
+    console.log(wallet, canMessage)
+    if (broadcasts_canMessage[i]) {
+      //If activated, start
+      const conversation = await xmtp.conversations.newConversation(wallet)
+      // Send a message
+      const sent = await conversation.send('gm')
+      console.log(sent.sent)
```
Comment (rygine) [inline README.md:365]: for this example code, WDYT of removing these `console.log` calls?

File: README.md [hunk 3]
```diff
@@ -320,7 +320,56 @@ for (const conversation of myAppConversations) {
...
+For example:
+
+```js
+const ethers = require('ethers')
+const { Client } = require('@xmtp/xmtp-js')
+
+//Rate limits
+//1k publish request /5 minutes / IP
+//publish request would be Client.create() , send() and newConversation()
+//10k of any request type / 5 mins / IP
```
Comment (rygine) [inline README.md:339]: what do these comments mean in the context of this code?

Comment (jhaaaa) [reply README.md]: Removed the comments - ty!

File: README.md (hunk 2 above)
Comment (jhaaaa) [reply README.md]: Removed the console.log calls - ty!

**Review by rygine — APPROVED** [review]

**Pull request merged**